src/index.html

```diff
@@ -171,12 +171,12 @@ <h3>New furniture</h3>
             </div>
             <div class="content">
               <h5>Aenean Ru Bristique</h5>
-              <div class="stars">
-                <a href="#" class="full"></a>
-                <a href="#" class="full"></a>
-                <a href="#"></a>
-                <a href="#"></a>
-                <a href="#"></a>
+              <div id="rate1" class="stars">
```

Comment:
Potrzebujemy te id? Z tego co widzialem i tak w kodzie JS tego potem nie wykorzystujemy.

Reply:
Usunięto id - w pierwszej wersji kodu były używane, potem znalazłam "sprytniejszy" sposób w js, ale już nie zupdatowałam html'a.